gravitee-apim-console-webui/src/management/api/apis.module.ts

```diff
@@ -98,6 +99,7 @@ import { AlertsModule } from '../../components/alerts/alerts.module';
     DocumentationModule,
     GioPolicyStudioRoutingModule,
     SpecificJsonSchemaTypeModule,
+    ApiHistoryV4Module,
```

Comment:
Can you put it up in the list, I think we tried to keep the alphabetical order 🤔

Reply:
no problemo, done